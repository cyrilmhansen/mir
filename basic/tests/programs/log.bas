10 PRINT LOG2(1)
20 PRINT LOG2(0)
30 PRINT LOG10(1)
40 PRINT LOG10(0)
50 PRINT LOG2(4)
60 PRINT LOG10(100)
70 END
