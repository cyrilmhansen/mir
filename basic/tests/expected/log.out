0
-Infinity
0
-Infinity
2
2
